### DISCUSSION TITLE
how to use these events

### DISCUSSION THREAD

**Discussion by joukhar** [GrapesJS/grapesjs#4598] [Q&A] [answered]
how to use these events plz
list of events : https://prnt.sc/loJtuv5L3S97

**Comment by artf** [comment, marked as answer]
Those events are component-related, so you should be able to use them inside your [custom components](https://grapesjs.com/docs/modules/Components.html#listen-to-property-changes)

**Reply by joukhar** [reply]
thanks for ur answer
